tests/baselines/reference/octalLiteralInStrictModeES3.errors.txt

```diff
@@ -1,8 +1,11 @@
 tests/cases/conformance/parser/ecmascript5/StrictMode/octalLiteralInStrictModeES3.ts(2,1): error TS1085: Octal literals are not available when targeting ECMAScript 5 and higher. Use the syntax '0o3'.
+tests/cases/conformance/parser/ecmascript5/StrictMode/octalLiteralInStrictModeES3.ts(2,1): error TS1121: Octal literals are not allowed in strict mode.
 
 
-==== tests/cases/conformance/parser/ecmascript5/StrictMode/octalLiteralInStrictModeES3.ts (1 errors) ====
+==== tests/cases/conformance/parser/ecmascript5/StrictMode/octalLiteralInStrictModeES3.ts (2 errors) ====
     "use strict";
     03;
     ~~
-!!! error TS1085: Octal literals are not available when targeting ECMAScript 5 and higher. Use the syntax '0o3'.
\ No newline at end of file
+!!! error TS1085: Octal literals are not available when targeting ECMAScript 5 and higher. Use the syntax '0o3'.
+    ~~
+!!! error TS1121: Octal literals are not allowed in strict mode.
```

Comment:
Seems undesirable to give a double-error here.

Reply:
The target < ES5 check seems weird as there is no strict mode in ES3, so I assume that’s a mistake and reversed the condition. I actually did wonder if we should completely remove the number literal octal check in the binder since this is already covered in the checker.